### PR TITLE
Add save for review forms, initial work 

### PR DESCRIPTION
**Pull request by ajnyga**
Initial work for https://github.com/pkp/pkp-lib/issues/3698, not ready for merge yet

### CODE REVIEW

File: pages/reviewer/ReviewerHandler.inc.php [hunk 1]
```diff
@@ -95,6 +95,24 @@ function _validateAccessKey($request) {
 			$this->user = $user;
 		}
 	}
+
+	/**
+	 * @copydoc PKPReviewerHandler::getReviewForm()
+	 */
+	public function getReviewForm($step, $request, $reviewerSubmission, $reviewAssignment) {
```
Comment (asmecher) [inline pages/reviewer/ReviewerHandler.inc.php]: This should only contain the `case 3`, I think, and the class should be `ReviewerReviewStep3Form`, not `PKPReviewerReviewStep3Form`.

Comment (ajnyga) [reply pages/reviewer/ReviewerHandler.inc.php]: ah yes.

If I just add case 3 then I should probably call for the parent function as well. I figured that almost the same amount of code if I just add all three cases?

Comment (asmecher) [reply pages/reviewer/ReviewerHandler.inc.php]: I agree, not a big deal either way; I'm slightly in favour of just overriding the one we care about in the application.

Comment (ajnyga) [reply pages/reviewer/ReviewerHandler.inc.php]: how about now?

**Comment by asmecher** [comment]
Merged manually (merge conflict)

**Pull request closed without merging**